arrays/arrays.c

```diff
@@ -69,11 +76,15 @@ void resize_array(Array *arr) {
  *
  * Throw an error if the index is out of range.
  *****/
-char *arr_read(Array *arr, int index) {
+char *arr_read(Array *array, int index) {
 
   // Throw an error if the index is greater than the current count
-
+  if (index > array->count)
```

Comment:
Good, you should also print an error message to standard error before you return on the next line.